src/System.Threading.Overlapped/tests/System.Threading.Overlapped.Tests.csproj

```diff
@@ -31,8 +31,12 @@
     <Compile Include="ThreadPoolBoundHandle_DisposeTests.cs" />
     <Compile Include="ThreadPoolBoundHandle_BindHandleTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'==''">
+    <Compile Include="OverlappedTests.cs" />
```

Comment:
overlapped is getting introduces for the first time, i don't see the need to append netstandard to it